services/requester/requester.go

```diff
@@ -255,16 +258,20 @@ func (e *EVM) signAndSend(ctx context.Context, script []byte, args ...cadence.Va
 	return flowTx.ID(), nil
 }
 
-func (e *EVM) GetBalance(ctx context.Context, address common.Address, height uint64) (*big.Int, error) {
-	// todo make sure provided height is used
+func (e *EVM) GetBalance(
```

Comment:
in all the implementations below I believe you should handle the case where the height is `LatestHeight` for which you should call the `ExecuteScriptAtLatestBlock`